doc/cookbook.rst

```diff
@@ -21,32 +21,34 @@ following is for one instrument in one working directory.
 
 1. Prepare your data
 
-    - Identify folder(s) with raw images
-    - The raw images can be gzip compressed
+  - Identify folder(s) with raw images
+  - The raw images can be gzip compressed
 
-2. Generate the instrument PYPIT reduction file (:doc:`pypit_file`) and setup file (:doc:`setups`)
+2. Run the pypit_setup script
 
-    - Run the pypit_setup script for the instrument
-    - Inspect the .setup file to confirm the instrument configurations
-    - Inspect the .group file to confirm calibrations exist
-    - Modify the data and spect blocks in .pypit file, as needed
-    - Rerun as needed
+    - Generates the instrument PYPIT reduction file (:doc:`pypit_file`)
+    - Generates the instrument .setups file (:doc:`setups`)
+    - Generates a custom PYPIT reduction file for each setup
+    - Inspect the .setups file to confirm the instrument configurations
+    - Modify the data and spect blocks in master .pypit file, as needed
+    - Rerun pypit_setup as needed
```

Comment:
Perhaps we should expand on "as needed" here. Should `pypit_setup` be rerun only if changes are made? Should this step be: `If any changes are made, you should rerun pypit_setup`